<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_dlg</class>
 <widget class="QDialog" name="client_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>491</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="log_tedt"/>
    </item>
    <item>
     <widget class="QPushButton" name="clear_btn">
      <property name="text">
       <string>&amp;Clear log</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connect_btn">
      <property name="text">
       <string>&amp;Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
